src/plugins/ExportKeras/ExportKeras.js

```diff
@@ -0,0 +1,73 @@
+/*globals define*/
+/*eslint-env node*/
+define([
+    'plugin/PluginConfig',
+    'text!./metadata.json',
+    'SimpleNodes/Constants',
+    'plugin/GenerateKeras/GenerateKeras/GenerateKeras'
+], function (
+    PluginConfig,
```

Comment:
Unused. (Code climate misses these since they are not the last argument to a function.)